docs/toc.jl

```diff
@@ -1,3 +1,3 @@
 TRANSITIONIDENTIFIERS_PAGES = [
-    "TransitionIdentifiers.jl" => "index.md",
+    "TransitionIndicators.jl" => "index.md",
```

Comment:
```suggestion
    "index.md",
```
Documenter automatically obrtains the sidebar title from the page title if they shouldn't be different.